.tekton/collector-slim-retag.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/stackrox/stackrox?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "500"
+    # TODO(ROX-21073): re-enable for all PR branches
+    pipelinesascode.tekton.dev/on-cel-expression: |
+      (event == "push" && target_branch.matches("^(master|release-.*)$")) ||
+      (event == "pull_request" && (source_branch.matches("(konflux|renovate|appstudio|rhtap)") || body.pull_request.labels.exists(l, l.name == "konflux-build")))
+  labels:
+    appstudio.openshift.io/application: acs
+    appstudio.openshift.io/component: collector-slim-retagged
+    pipelines.appstudio.openshift.io/type: build
+  name: collector-slim-retagged-on-push
+  namespace: rh-acs-tenant
+
+spec:
+
+  params:
+  - name: git-url
+    value: '{{source_url}}'
+  - name: revision
+    value: '{{revision}}'
+  - name: output-image-repo
+    value: quay.io/rhacs-eng/collector-slim
+  - name: input-image-repo
+    value: quay.io/rhacs-eng/collector
```

Comment:
We build and push slim as `quay.io/rhacs-eng/collector:<blah>-fast-slim`, ref https://github.com/stackrox/collector/blob/f1748e6301357bf28d648f8a4ba253b95e06cd90/.tekton/collector-component-pipeline.yaml#L422

Since currently "slim" is the same as "full", this pipeline retags "full" into the "slim" repo. No harm until "slim" and "full" would differ again which, I believe, is not planned to happen. On the contrary, "slim" is planned to disappear.

`quay.io/rhacs-eng/collector:<blah>-slim` was a StackRox convention set _not by us_ long time ago.
CPaaS did not allow suffixes in tags and so we had to create a separate repo for _releasing_ "slim".
In the process of building that out we came up with an idea to retag Scanner and Collector images _upstream_ and so `-slim` should have disappeared from the suffix, that's how `quay.io/rhacs-eng/collector-slim` repo was born.

Not sure to what extent to describe this but I tried.